oxide.go

```diff
@@ -33,25 +33,25 @@ const (
 )
 
 const (
-	flagHost                      = "oxide-host"
-	flagToken                     = "oxide-token"
-	flagProject                   = "oxide-project"
-	flagVCPUs                     = "oxide-vcpus"
-	flagMemory                    = "oxide-memory"
-	flagBootDiskSize              = "oxide-boot-disk-size"
-	flagBootDiskImageID           = "oxide-boot-disk-image-id"
-	flagVPC                       = "oxide-vpc"
-	flagSubnet                    = "oxide-subnet"
-	flagUserDataFile              = "oxide-user-data-file"
-	flagSSHUser                   = "oxide-ssh-user"
-	flagAdditionalSSHPublicKeyIDs = "oxide-additional-ssh-public-key-ids"
+	flagHost                      = "host"
+	flagToken                     = "token"
+	flagProject                   = "project"
+	flagVCPUs                     = "vcpus"
+	flagMemory                    = "memory"
+	flagBootDiskSize              = "boot-disk-size"
+	flagBootDiskImageID           = "boot-disk-image-id"
+	flagVPC                       = "vpc"
+	flagSubnet                    = "subnet"
+	flagUserDataFile              = "user-data-file"
+	flagSSHUser                   = "ssh-user"
+	flagAdditionalSSHPublicKeyIDs = "additional-ssh-public-key-ids"
```

Comment:
The flags must remain as `[DRIVER]-[FLAG-NAME]` (e.g., `oxide-host`). Otherwise Rancher will not download the machine driver and will log the following error.

```
2024/12/05 00:21:11 [INFO] Copying management-state/machine-drivers/1b6eae9d7ef9d359779d42e4763ce1333f614bb25af5dae23733c3e8e9bc0d53-docker-machine-driver-oxide => /opt/drivers/management-state/bin/docker-machine-driver-oxide-tmp
2024/12/05 00:21:11 [ERROR] error syncing 'oxide': handler node-driver-controller: parameter host does not follow expected naming convention [DRIVER]-[FLAG-NAME], requeuing
```